### DISCUSSION TITLE
迅雷云盘挂ALLISE问题

### DISCUSSION THREAD

**Discussion by JianNY13** [AlistGo/alist#3827] [Q&A]
迅雷云盘挂alist，每次都要重新获取验证码，重新填写才能使用，有一次搞定的方法吗？

**Comment by KennyZKY** [comment]
我专家版，用的用户名登录，正常

**Comment by JianNY13** [comment]
谢谢你啦




怼沵、憅杺
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月12日(星期天) 晚上10:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] 迅雷云盘挂ALLISE问题 (Discussion #3827)





 
我专家版，用的用户名登录，正常

**Comment by JianNY13** [comment]
专家版 挂载好复杂哦&nbsp;&nbsp;






怼沵、憅杺
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月12日(星期天) 晚上10:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] 迅雷云盘挂ALLISE问题 (Discussion #3827)





 
我专家版，用的用户名登录，正常
